### ISSUE TITLE
Firefox Nightly 58 - PopUp menu grapgical bug

### ISSUE THREAD

**Issue by Acid-Crash** [freaktechnik/advanced-github-notifier#56]
On Firefox Nightly 58 Pop-Up menu seems kinda messed up.
![gitpopup](https://user-images.githubusercontent.com/32600318/32558492-879fb2c0-c4ad-11e7-81e1-39f755a0924d.png)
Looks like Hover styling isn't removed correctly.
This appears when hovering over each item and then stopping you hover over "Mark All as read" or "Show all notification"

**Comment by freaktechnik** [comment]
Yeah... I should probably write my own styles either way, since those styles aren't technically supported anymore. Also improves the situation with #49

**Comment by Acid-Crash** [comment]
Just a small clarification: it would be great if the future styling could match the one from original Firefox57+ popUp menus (aka Bookmark Menu, or Overflow Menu)

**Comment by freaktechnik** [comment]
It would be my goal to stay close to a native look.

**Comment by Acid-Crash** [comment]
I may be mistaking but here are the colors for that menu (default theme).
item:hover -moz-dialog;
separator - hsla(210,4%,10%,.20)
ShowAll - hsla(0,0%,80%,.3)
ShowAll:hover - hsla(0,0%,80%,.45)

**Comment by freaktechnik** [comment]
It should probably end up using https://www.npmjs.com/package/photon-colors the style guide etc.
